sopel/bot.py

```diff
@@ -244,6 +244,10 @@ def register(self, callables, jobs, shutdowns, urls):
         for func in urls:
             self.memory['url_callbacks'][func.url_regex] = func
 
+    def invite(self, nick, channel):
+        """Invite nickname to the bots channel"""
+        self.write(['INVITE', nick, channel])
```

Comment:
You can simply use `bot.write(['INVITE', trigger.group(3), trigger.group(4)])` below for the same effect without adding a function to core.